symplyphysics/laws/relativistic/vector/acceleration_via_force.py

```diff
@@ -0,0 +1,93 @@
+from sympy import sqrt, symbols, sympify
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    validate_input,
+    validate_output,
+    Vector,
+    scale_vector,
+    QuantityVector,
+    add_cartesian_vectors,
+    dot_vectors,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.relativistic.vector import force_via_acceleration
+
+# Description
+# In special relativity, the Newton's second law does not hold in the form `F = m * a`. There still exists a relation
+## between force `a` and acceleration `F`, albeit more complex.
+
+# Law: a = 1 / (m0 * gamma) * (F - dot(v, F) * v / c**2)
+## a - vector of acceleration
+## m0 - rest mass
+## v - vector of velocity
+## c - speed of light
+## F - vector of force
+## gamma = 1 / sqrt(1 - dot(v, v) / c**2) - Lorentz factor
+## dot(a, b) - dot product between vectors `a` and `b`
+
+# Notes
+## - For force and rest mass expressions, see [force via acceleration law](./force_via_acceleration.py)
```

Comment:
It is the same law so no need for separate description. We only have to mention, that `a` is made of two components. Give it a try and you will see that combined together it will look cleaner.